db/migrate/20210918120010_create_users.rb

```diff
@@ -0,0 +1,19 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :email
+      t.string :user_name
```

Comment:
What do you think about `username` or `nickname`?

Reply:
user and name are two words joined by ruby style guide underscore, `nickname` is not suitable